### ISSUE TITLE
Turn fixed heating circuits on and off

### ISSUE THREAD

**Issue by andyx87** [signalkraft/mypyllant-component#70]
### Before submitting a new issue

- [ X ] I'm not using the older Vaillant sensoAPP (which is unsupported by this integration)
- [ X ] I [redownloaded the latest release](https://my.home-assistant.io/redirect/hacs_repository/?owner=signalkraft&repository=mypyllant-component) and restarted Home Assistant
- [ X ] I've [enabled debug logs](https://signalkraft.com/mypyllant-component/3-contributing/#debugging) and attached the relevant logs to this ticket

### Problem description

It's not a problem, but a suggestion for improvement

When I looked at the test data, I noticed that the two virtual fixed heating circuits can also be seen there. It would be great if you could turn these on and off. This would make it possible to increase the temperature in the buffer storage (cylinder) without increasing the target flow temperatures in the heating circuit. This way you could store energy if there is leftover PV electricity or if the price of electricity is cheap (Tibber).

**Comment by andyx87** [comment]
Not until now

**Comment by boelle** [comment]
@signalkraft i would also vote for this one, 

the way i would do it would be to use the nordpool integration to figure when electricity is cheap and fill up the cylinder when its cheap, and then monitor solar and if i use less electricity than i generate then first fill up the cylinder and if that is full then export

but export is almost free energy to the grid as they pay you less and less these days

but it would only be doable if you can control the virtual circuits


for my own case i only have a hotwater cylinder and then a buffer tank for the house heating, that one is used to pull energy from for de-icing the outside unit, mostly in the winter where condensate freeze up the unit, since its plumbed directly to heating i can not control it. i just use the heating curve and the minimum temp controls to make sure the circuit does not go so low it stops heating (saftety feature)

**Comment by boelle** [comment]
@signalkraft poke
